cypress/integration/project/checkout.spec.ts

```diff
@@ -106,25 +106,6 @@ context("project checkout", () => {
               expect(result.stdout).to.equal(`rad`);
             }
           );
-
-          // Make sure we can't check out a project to the same directory twice.
```

Comment:
Is this removed because the behavior changed?

Reply:
We saw that `checkout` is idempotent, as confirmed by this test in `working_copy.rs` https://github.com/radicle-dev/radicle-upstream/pull/1463/files#diff-940a798aab5c5c9a4058c1e8c8b384b600a22e234ece99249c8bd028a5be74dbR44

Reply:
Thanks for the thoroughness @rudolfs :bow: So what's happening is that the project with the same name and description has a different URN to the same name and _no description_. This is because they have different hash outputs. So, with this in mind, it's probably best that we don't allow to use the same working copy for checkout and guard against it.

Reply:
Actually, this is a bit more sinister than I thought. Before we were able to tell if an `Entity` ended up with the same `URN`. But we can only know the URN once we create the project, which is idempotent. So we can't easily tell if the project was created or just didn't do anything. Need to think about that more.

Reply:
> Before we were able to tell if an `Entity` ended up with the same `URN`.

Wait what, how is this different now?

Reply:
Is it possible to determine the `Urn` before calling `create`? :thinking:

Reply:
In theory yes: https://github.com/radicle-dev/radicle-link/blob/d5f2e37cb6a751c6f8e18bc1c977ad27d4e9c8f5/librad/src/identities/git.rs#L485

Reply:
Right, was just looking at that :sweat_smile: I'll create a proof for theory tomorrow